features/support/page_objects/step_six_page.rb

```diff
@@ -3,6 +3,7 @@ class StepSixPage < BasePage
     element :benefits_text, '.text', text: 'contact the Department for Work and Pensions'
     element :benefit, '.text'
     elements :li, 'li'
+    elements :td, 'td'
```

Comment:
The page object pattern is supposed to provide a nice interface to the test suite that isolates it from the implementation's details like td's this etc...  So having a method called :td just to give you all td's seems to be defeating the object of it.  But, there seems to be a pattern set down already with the :li above it.

Reply:
Yeah I know. The problem is the element doesn't have a descriptive name. If you change it to one then you add a layer of ambiguity. I think it's fine as td